frontend/integration-tests/tests/event.scenario.ts

```diff
@@ -0,0 +1,58 @@
+import { browser, $, element, by, ExpectedConditions as until } from 'protractor';
+import { appHost, checkLogs, checkErrors, testName } from '../protractor.conf';
+import { safeLoad, safeDump } from 'js-yaml';
+import * as _ from 'lodash';
+import * as crudView from '../views/crud.view';
+import * as yamlView from '../views/yaml.view';
+
+describe('Events', () => {
+  const leakedResources = new Set<string>();
```

Comment:
`leakedResources`?

Reply:
Copy and pasting from another test. Cleaning it up.